includes/sanitizers/class-amp-iframe-sanitizer.php

```diff
@@ -192,4 +192,28 @@ private function build_placeholder( $parent_attributes ) {
 
 		return $placeholder_node;
 	}
+
+	/**
+	 * Removes a child of a node.
+	 *
+	 * Also, calls the mutation callback for it.
+	 * This tracks all the nodes that were removed.
+	 *
+	 * @since 0.7
+	 *
+	 * @param object $child The node to remove.
+	 * @param object $parent The parent node for which to remove the child (optional).
```

Comment:
Instead of `object` these should probably be `DOMNode`.

Also, is the `$parent` even needed since it will always be available via `$child->nodeParent`?

Reply:
Thanks, @westonruter. This [commit](https://github.com/Automattic/amp-wp/pull/912/commits/90090b0727cd652c197be8391bb21f8d72614793) changes the `@param` type to `DOMNode`, and removes the second parameter. Thanks for pointing out that the second isn't needed.